src/components/NavToggle/styles.scss

```diff
@@ -0,0 +1,78 @@
+// Mobile first
+
+.wrapper {
+		display: inline-flex;
+		position: fixed;
+		top: 1.15em;
+		right: 1em;
+		align-items: center;
+		font-size: 1.5rem;
+		
+		&::before {
+			content: "";
+			display: block;
+			width: 1em;
+			height: 1.15em;
+			font-size: 1.33em;
+			background: url(images/icons/cross.svg) center / contain no-repeat;
+			background-color: green;
+		}
+	}
+
+	.container {
+		display: inline-block;
+		position: relative;
+		margin-left: 0.8em;
+		font-family: var(--font_family__body);
+		text-transform: uppercase;
+	}
+
+		.label {
+			&[aria-hidden="true"]{
+				opacity: 0;
+			}
+
+			&[aria-hidden="false"]{
+				opacity: 1;
+			}
+
+			&.open {
+				position: absolute;
+				top: 0;
+				right: 0;
+			}
+		}
+
+// Mobile landscape
+@media (orientation: landscape) {
+	.wrapper {
+		font-size: 1rem;
+	}
+}
+
+// Tablet portrait
+@media (min-width: 768px) {
+	.wrapper {
+		right: 50%;
+		font-size: 2rem;
+		transform: translateX(50%);
```

Comment:
This works, but I would default to
```
left: 50%;
transform: translateX(-50%);
```
as your go-to solution to this kinda thing, as the convention is to think from left -> right, but yeah - reality is that it works either way.